src/content/en/tools/lighthouse/audits/budgets.md

```diff
@@ -0,0 +1,283 @@
+project_path: /web/tools/_project.yaml
+description: Official documentation for the "Performance Budgets" and "Keep Request Counts And File Sizes Small" Lighthouse audits.
+book_path: /web/tools/_book.yaml
+
+{# wf_updated_on: 2019-05-05 #}
+{# wf_published_on: 2019-04-30 #}
+{# wf_blink_components: Platform>DevTools #}
+
+# Performance Budgets (Keep Request Counts And File Sizes Small) {: .page-title }
+
+## Overview {: #overview }
+
+<aside class="note">
+  The <b>Performance Budgets</b> and <b>Keep Request Counts And File Sizes Small</b> audits both
```

Comment:
"Keep Request Counts And File Sizes Small" --> "Keep Request Counts Low And File Sizes Small" here and below

Reply:
Done